### ISSUE TITLE
License issues/incompatibilities

### ISSUE THREAD

**Issue by HarshedCodes** [MrMKN/Professor-bot#119]
Hey there, just nothing some issues for you. Your repo says it's licensed under the AGPL, but then links to the GPL2.

Additionally, it contains restrictions ("selling this code is STRICTLY PROHIBITED") that is incompatible with either the GPL or AGPL.

As explained by the authors of that license:
'If a license does not permit users to make copies and sell them, it is a nonfree license.'
https://www.gnu.org/philosophy/selling.en.html

In short, your repo claims it's under two different licenses that are not two-way compatible and then additionally implicitly asserts it's under some third license that's incompatible with both of the first two options.

**Comment by MrMKN** [comment]
Problem Solved ✓🙂
